tests/searchfox-config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "mozsearch_path": "$MOZSEARCH_PATH",
+
+  "default_tree": "searchfox",
+
+  "trees": {
+    "searchfox": {
+      "index_path": "$WORKING/searchfox",
+      "files_path": "$MOZSEARCH_PATH",
```

Comment:
Why the change to `files_path` here? Does this change mean that [this line](https://github.com/mozsearch/mozsearch/blob/ab35fab17cc1d78975094530fb15884a6149944a/tests/searchfox/setup#L8) can be removed?

Reply:
I think I experienced a patherror, said something to myself like "how did this ever work?" without realizing that line existed, and just changed it.  I'll try removing the line and seeing what happens, etc.

Thanks for all the eagle-eye reviews!  Your diligence definitely pays off on these "fancy" path extractions!

Reply:
Removing the line worked fine.  I'm going to push with this fix.